File: Iniciante/PHP/exercise_1048.php
<?php

function setNovoSalario($salario) {
    $novoSalario = $salario;
    $reajuste = 0;
    $porcentagemReajuste = 0;

    switch (true) {
        case ($salario >= 0.0 && $salario <= 400.0):
            $porcentagemReajuste = 0.15;
            $reajuste=$salario*$porcentagemReajuste;
            $novoSalario+=$reajuste;
            break;
        case ($salario > 400.0 && $salario <= 800.0):
            $porcentagemReajuste = 0.12;
            $reajuste=$salario*$porcentagemReajuste;
            $novoSalario+=$reajuste;
            break;
        case ($salario > 800.0 && $salario <= 1200.0):
            $porcentagemReajuste = 0.10;
            $reajuste=$salario*$porcentagemReajuste;
            $novoSalario+=$reajuste;
            break;
        case ($salario > 1200.0 && $salario <= 2000.0):
            $porcentagemReajuste = 0.07;
            $reajuste=$salario*$porcentagemReajuste;
            $novoSalario+=$reajuste;
            break;
        case ($salario > 2000):
            $porcentagemReajuste = 0.04;
            $reajuste=$salario*$porcentagemReajuste;
            $novoSalario+=$reajuste;
            break;
    }
    return ['novo_salario' => $novoSalario, 'porcentagem_reajuste' => $porcentagemReajuste, 'reajuste' => $reajuste];
}

$salario = floatval(fgets(STDIN));

$novoSalario = setNovoSalario($salario);

print_r("Novo salario: ".str_replace(',', '', number_format($novoSalario['novo_salario'], 2))."\n");
print_r("Reajuste ganho: ".str_replace(',', '', number_format($novoSalario['reajuste'], 2))."\n");
print_r("Em percentual: ".($novoSalario['porcentagem_reajuste']*100)." %\n");

//Concluído

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1048.py
def calcularValorReajuste(salario):
    if salario >= 0 and salario <= 400:
        return ['15', salario * 0.15]
    elif salario > 400 and salario <= 800:
        return ['12', salario * 0.12]
    elif salario > 800 and salario <= 1200:
        return ['10', salario * 0.1]
    elif salario > 1200 and salario <= 2000:
        return ['7', salario * 0.07]
    else:
        return ['4', salario * 0.04]

salarioAtual = float("{:.2f}".format(float(input())))
porcentagemReajuste, valorReajuste = calcularValorReajuste(salarioAtual)
salarioNovo = salarioAtual + valorReajuste

print(''.join(['Novo salario: ', "{:.2f}".format(salarioNovo)]))
print(''.join(['Reajuste ganho: ', "{:.2f}".format(valorReajuste)]))
print(''.join(['Em percentual: ', str(porcentagemReajuste), ' %']))